src/owr/OptimisticWithdrawalRecipientV2.sol

```diff
@@ -370,31 +380,22 @@ contract OptimisticWithdrawalRecipientV2 is Clone, OwnableRoles {
 
     // load storage into memory
     uint256 currentbalance = address(this).balance;
-    uint256 _claimedPrincipalFunds = uint256(claimedPrincipalFunds);
     uint256 _memoryFundsPendingWithdrawal = uint256(fundsPendingWithdrawal);
     uint256 _fundsToBeDistributed = currentbalance - _memoryFundsPendingWithdrawal;
 
-    (address principalRecipient, address rewardRecipient, uint256 amountOfPrincipalStake) = getTranches();
-
-    // determine which recipeint is getting paid based on funds to be
-    // distributed
+    // determine which recipeint is getting paid based on funds to be distributed
     uint256 _principalPayout = 0;
     uint256 _rewardPayout = 0;
 
     unchecked {
```

Comment:
what does unchecked do on a block of code thats only boolean/comparison and assignments. Are the greater/thans less/thans doing safe math under the hood and we're turning that off?

Reply:
I think it was an attempt to save gas, because `unchecked` disables overflow checks if, in according to our logic, we can guarantee this never overflows.

Specifically 
```
          // shouldn't underflow
          _rewardPayout = _fundsToBeDistributed - amountOfPrincipalStake;
```

However, I don't think it worth it. Left it intact relying on the previous audit that accepted it.